src/VolleyManagement.Backend/VolleyManagement.UI/Areas/Mvc/Views/Tournaments/_PlayoffGameResult.cshtml

```diff
@@ -1,18 +1,28 @@
 ﻿@model VolleyManagement.UI.Areas.Mvc.ViewModels.GameResults.GameResultViewModel
 
+<link rel="stylesheet" href="~/Content/Site.css" />
+
 <div class="table-bordered">
 
     @if (!Model.HomeTeamId.HasValue && !Model.AwayTeamId.HasValue)
     {
-        <div class="row">
-            <div class="col-md-10">
-                <strong>  @Resources.UI.TournamentViews.HomeTeamPlaceholder </strong>
-            </div>
-        </div>
-
-        <div class="row">
-            <div class="col-md-10">
-                <strong>  @Resources.UI.TournamentViews.AwayTeamPlaceholder </strong>
+        <div class="container-fluid">
+            <div class="row">
+                <div class="col-md-3 game-number">
+                    <div>
+                        <strong>@Html.Raw("G" + @Model.GameNumber) </strong>
```

Comment:
Such formatting is a responsibility of view model

Reply:
Should I change int to string and add "G" in property( public int GameNumber { get; set; }) or add new property FormatGameNumber(for example)?